src/ADNLPProblems/triangle.jl

```diff
@@ -0,0 +1,59 @@
+include("../../data/triangle.jl")
+export triangle
+
+function triangle(x0 = xe, TRIS::Vector{Int64} = Tr, Const::Vector{Int64} = Constants; n::Int = default_nvar, type::Val{T} = Val(Float64), kwargs...) where {T}
+  τ = zero(T)
```

Comment:
Found what was missing ! You also need to add
```
x0 = T.(x0)
```
so that `x0` wlll be of type `T`.